web/src/beta/features/Editor/useStorytelling.ts

```diff
@@ -42,6 +43,21 @@ export default function ({ sceneId }: Props) {
     [selectedStory?.pages, selectedStoryPageId],
   );
 
+  const onTimeChange = useCallback(
```

Comment:
If you use this onTimeChange in the right panel like you are doing, this is fine.
But you also pass this in to the Visualizer, which is strange. 
Use of the visualizerRef should be to use features inside of core/Visualizer that you normally wouldn't be able to use outside. But setting upthis function and then pass it to the Visuzlier doesn't make much sense.